activerecord/lib/active_record/migration/compatibility.rb

```diff
@@ -16,32 +16,13 @@ def self.find(version)
       V6_1 = Current
 
       class V6_0 < V6_1
-      end
-
-      class V5_2 < V6_0
         module TableDefinition
-          def timestamps(**options)
-            options[:precision] ||= nil
+          def references(*args, **options)
+            options[:_uses_legacy_reference_index_name] = true
             super
           end
```

Comment:
```suggestion
          end
          alias :belongs_to :references
```

We'll need to support [`add_reference` and `add_belongs_to`](https://github.com/rails/rails/blob/28a4d973318092ffe74a4e4ecbe8794a38dc6c99/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#L970-L973) as well.